### PR TITLE
Completed css-ecommerce-page. Font looks different but I saw that ther…

### PR DESCRIPTION
**Pull request by sharon-tieu**
…e were no custom fonts for me to use in the files. Will wait for further instructions if sans-serif does not work
Screen Shot 2022-05-19 at 8 34 16 AM
.

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Sharon! This is the bit you missed about the fonts:
image

**Comment by sharon-tieu** [comment]
Screen Shot 2022-05-24 at 10 24 45 AM

File: css-ecommerce-page/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      </div>
+      <div class="col-one-fifth">
+        <img src="/css-ecommerce-page/images/dress2.jpeg" alt="Reformation dress image" class="image-5">
+        <p class="text-title-black">Reformation <span class="text-title-grey"> | Size 4</span></p>
+        <p><span class="text-title-black highlight-price">&#36;20</span><span class="strike-through-text">&#36;198 retail</span></p>
+      </div>
+    </div>
+    <div class="row-footer">
+
+    </div>
```
Comment (codingcodymiller) [inline css-ecommerce-page/index.html]: There should be no need for empty rows ever.

File: css-ecommerce-page/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+<body>
+  <div class="container">
+    <div class="row-header">
+      <div class="col-one-whole">
+        <h4>New Arrivals</h4>
+      </div>
+    </div>
+    <div class="pics-in-a-row">
+      <div class="col-one-fifth">
+        <img src="/css-ecommerce-page/images/dress1.jpeg" alt="Free People Dress image" class="image-1">
```
Comment (codingcodymiller) [inline css-ecommerce-page/index.html]: No root paths please.

File: css-ecommerce-page/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  <link href="/css-ecommerce-page/styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row-header">
+      <div class="col-one-whole">
+        <h4>New Arrivals</h4>
+      </div>
+    </div>
+    <div class="pics-in-a-row">
```
Comment (codingcodymiller) [inline css-ecommerce-page/index.html]: One `row` class, not individually named `row` classes.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by sharon-tieu** [comment]
Screen Shot 2022-05-27 at 10 55 49 PM

File: css-ecommerce-page/styles.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  padding: 10px;
+}
+
+.row {
+  text-align: center;
+  padding: 5px;
+  font-family:
+    "DM Sans",
+    sans-serif;
+  src: url("https://fonts.gstatic.com/s/dmsans/v11/rP2Hp2ywxg089UriCZ2IHTWEBlwu8Q.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css]: This `src` property is only necessary for the `@font-face` rules we provided for you. The only property you need to set a font family is `font-family`

File: css-ecommerce-page/styles.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
+.container {
+  padding: 10px;
+}
+
+.row {
+  text-align: center;
+  padding: 5px;
+  font-family:
+    "DM Sans",
+    sans-serif;
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css]: The only styling on the `row` class should be `display: flex`. You have that applied to the `display` class rule below this. Please revise this to follow proper layout classes protocol.

File: css-ecommerce-page/styles.css [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  display: flex;
+  position: relative;
+  width: 100%;
+  padding-bottom: 10px;
+  padding-left: 10px;
+  padding-right: 10px;
+  justify-content: center;
+}
+
+.col-on-fifth {
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css]: Is this supposed to be `col-one-fifth`? If so, its only styling should be `flex-basis: 20%`

File: css-ecommerce-page/styles.css [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+.strike-through-text {
+  text-decoration: line-through;
+  font-weight: lighter;
+  color: #e8e8e8;
+  font-family:
+    "DM Sans",
+    sans-serif;
+  src: url("https://fonts.gstatic.com/s/dmsans/v11/rP2Hp2ywxg089UriCZ2IHTWEBlwu8Q.woff2") format("woff2");
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css]: Not necessary

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by sharon-tieu** [comment]
Screen Shot 2022-06-11 at 1 05 50 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**